imaplugin/src/main/java/com/kaltura/playkit/plugins/imadai/IMADAIPlugin.java

```diff
@@ -1181,7 +1181,7 @@ public void onEvent(PlayerEvent.DurationChanged event) {
             if (mediaConfig.getMediaEntry() != null && mediaConfig.getMediaEntry().getMediaType() == PKMediaEntry.MediaEntryType.DvrLive) {
                 return PKMediaEntry.MediaEntryType.DvrLive;
             } else {
-                return PKMediaEntry.MediaEntryType.Live;
+                return PKMediaEntry.MediaEntryType.DvrLive; // need to make it LiveDvr so resume will continue from lasr pos.
```

Comment:
If stream is not DVR then this is incorrect, stream should resume from last live position, wether it’s inside ad or after it.